packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -1325,12 +1257,16 @@ export class ConversationService {
       clientsToRemove.qualified_user_map,
     );
 
-    const commitBundle = await coreCryptoClient.removeClientsFromConversation(
+    const commitBundle = await this.mlsService.removeClientsFromConversation(
       groupIdDecodedFromBase64,
       fullyQualifiedClientIds,
     );
 
-    const messageResponse = await this.sendCommitBundleRemovalMessages(groupIdDecodedFromBase64, commitBundle);
+    const messageResponse = await this.mlsService.uploadCoreCryptoCommitBundle(groupIdDecodedFromBase64, commitBundle);
```

Comment:
Let's merge those 2 methods in a single one and make  the other private so that it doesn't leak the coreCrypto bundle and this stays internal implementation details :)

Reply:
I've done it for both: `updateKeyingMaterial` and `removeClientsFromConversation` methods 👍